### PR TITLE
Add some github actions and a test

### PR DESCRIPTION
**Pull request by arnfinn**
- Read `csv` with `stringsAsFactors = FALSE`
- Added a test that reproduce bug #75, if it reappears.
- Activated Github Actions
- Removed appveyor

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    branches:
+      - master
+
+name: test-coverage
+
+jobs:
+  test-coverage:
+    runs-on: macOS-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```
Comment (areedv) [inline .github/workflows/coverage.yml:15]: Can the name "GITHUB_TOKEN" be used? Not set yet

File: tests/testthat/test-diabetes.R [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Test created because of issue #75
+# https://github.com/mong/imongr/issues/75
+#
+# Will fail until this bug has been fixed
+
+test_that("Test csv_to_df() with file", {
```
Comment (areedv) [inline tests/testthat/test-diabetes.R:6]: ```test_that()``` combined with the text should become a meaningful sentence describing what is being tested...

**Review by areedv — APPROVED** [review]
See also comments

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (arnfinn) [reply .github/workflows/coverage.yml:15]: Seems like it is there by default:
https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow

File: tests/testthat/test-diabetes.R (hunk 2 above)
Comment (arnfinn) [reply tests/testthat/test-diabetes.R]: fixed

**Pull request merged**